draft.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+set -euo pipefail
+
+function usage {
+    cat <<EOF
+usage: $0 <search-term>
+    Copies the flakefinder daily reports from the google cloud storage,
+    greps them for the search term and creates a list of urls leading
+    to the reports
+    Example:
+    > $0 test_id:1632
+    Requirements: 
+    * gsutil installed
+      https://cloud.google.com/storage/docs/gsutil_install
+EOF
+}
+
+function check_gsutil_installed {
+    if ! command -V gsutil > /dev/null; then
+        usage
+        echo 'gsutil is not installed'
+        exit 1
+    fi  
+}
+
+function check_args {
+    if [ $# -lt 1 ]; then
+        usage
+        exit 1
+    fi  
+}
+
+function main {
```

Comment:
Have you committed this by mistake?

Reply:
oh
yes,
I don't know what this is.
Sorry for that.